### PR TITLE
add sdk example for cookie monitoring

### PR DESCRIPTION
**Pull request by bendoyon**

### CODE REVIEW

File: docs/guides/cookie-monitoring.mdx [hunk 1]
```diff
@@ -44,7 +44,46 @@ These are end-to-end examples in various programming languages.
...
 <Accordion title="Python SDK Example">
 ```python
-# TODO
+import os
+import time
+
+from flareio import FlareApiClient
+
+
+api_key: str | None = os.environ.get("FLARE_API_KEY")
```
Comment (aviau) [inline docs/guides/cookie-monitoring.mdx:53]: Can you remove this in favour of:
```
api_client = FlareApiClient(
    api_key=os.environ["FLARE_API_KEY"],
)
```

Which provides a similar error and is simpler ?

File: docs/guides/cookie-monitoring.mdx [hunk 2]
```diff
@@ -44,7 +44,46 @@ These are end-to-end examples in various programming languages.
...
+
+    resp_data: dict = resp.json()
+
+    fetched_pages += 1
+    num_results: int = len(resp_data["items"])
+    print(f"Fetched page {fetched_pages} with {num_results} results...")
+
+    # Save the last "next" value.
+    last_from = resp_data.get("next") or last_from
+
```
Comment (aviau) [inline docs/guides/cookie-monitoring.mdx:81]: Can we add better guidance here? I am not sure "get the full data" is the best explaination ( I know where you copied this from!)

```python
    # Iterate on all cookies
    for cookie in resp_data["items"]:
      if cookie["name"] == "session-token":
        # Invalidate the cookie here
        print(f"Would invalidate cookie {cookie['value']}...")
```

**Review by aviau — APPROVED** [review]

**Comment by aviau** [comment]
Small comments but overall lgtm. Let's ship this fast!

File: docs/guides/cookie-monitoring.mdx (hunk 2 above)
Comment (bendoyon) [reply docs/guides/cookie-monitoring.mdx:81]: I went with `Iterate on cookies in the current page`

**Pull request merged**